### ISSUE TITLE
GHC output reports "Invalid option `--stdout'"

### ISSUE THREAD

**Issue by neilco** [dramforever/vscode-ghc-simple#54]
## Status

- [X] A workaround is available: Upgrade stack to at least 2.1.1. Newest is 2.1.3 at time of writing.
- [ ] A fix is written
- [ ] A fix is released
- [ ] The fixed version is on the marketplace.

---

- Version of vscode-ghc-simple: 0.1.18
- Version of VSCode: 1.43.2
- Version of Haskell-related tools:
    
    - GHC: 8.4.4
    - Stack: 1.9.3

## Steps to reproduce

- Install the extension
- Open a project

## Workspace

It is a private repository and I cannot share any other details. 
## GHC log

```shell
Error checking [REDACTED]: Command stack ide targets failed:
Invalid option `--stdout'

Usage: stack ide targets [--help]
  List all available stack targets
```

**Comment by dramforever** [comment]
Sorry about that breakage. (Details in rest of paragraph) The old `stack ide targets` command prints error messages along with 'real' output to stderr, making it hacky to get the output, so when I learned that it was possible to have it switch to stdout, I just used it. Apparently I didn't consider that it broke pre-2.1.1 users. This is a oversight on my part.

I'll probably put the original behavior back since it does work, and also probably document version requirements and put in checks after I do determine the actual needed versions.

But meanwhile if your setup is compatible (it's private, but I don't see why it wouldn't be), just upgrade to the latest version of stack. Yours is nearly a year old now (assuming you meant 1.9.3).
